components/domActions.js

```diff
@@ -0,0 +1,85 @@
+import createButton from "../utilities/createButton.js";
+import emptyTodoError from "../utilities/emptyTodoError.js";
+import timeCalculation from "../utilities/TimeCalculation.js";
+import { createTodoDiv, textArea } from "./domElements.js";
+import { todoWrapper } from "./domElements.js";
+
+export const openCreateTodo = () => {
+  createTodoDiv.classList.add('open-create-todo');
+  textArea.focus();
+}
+
+export const cancelTodo = () => {
+  createTodoDiv.classList.remove('open-create-todo');
+  textArea.value = '';
+}
+
+let todos = [];
+
+export const renderTodoList = () => {
+  const task = textArea.value;
+
+  const todo = {
+    title: task,
+    isDone: false,
+    isEditing: false,
+    createdAt: timeCalculation(),
+  }
+
+  todos.push(todo);
+
+  textArea.value = '';
+  emptyTodoError('Successfully created new todo');
+
+  return todos;
+};
```

Comment:
`renderTodoList` is supposed to be adding actual DOM elements to the document.
Let's rename the function to `createTodoObject`
```suggestion
export const createTodoObject = () => {
  const task = textArea.value;

  const todo = {
    title: task,
    isDone: false,
    isEditing: false,
    createdAt: timeCalculation(),
  }
};
```

Notice that we wouldn't do anything else in this function following Single Responsibility principle.